chains/pulsechain/metadata.yaml

```diff
@@ -1,11 +1,15 @@
 blockExplorers:
   - apiUrl: https://api.scan.pulsechain.com/api
     family: blockscout
-    name: PulseChain Explorer
-    url: https://scan.pulsechainfoundation.org
+    name: Pulsechain Explorer
+    url: https://oldscan.gopulse.com/#/
+  - apiUrl: https://api.scan.pulsechain.com/api
+    family: blockscout
```

Comment:
Is this one actually blockscout-based? Doesn't seem like it from the UI. If not, set this to `other`